prow/BUILD.bazel

```diff
@@ -130,6 +130,7 @@ filegroup(
         "//prow/labels:all-srcs",
         "//prow/logrusutil:all-srcs",
         "//prow/metrics:all-srcs",
+        "//prow/monitoring:all-srcs",
```

Comment:
@stevekuznetsov @munnerz Any idea why this got added here and why `prow/monitoring/BUILD.bazel` got changed? I haven't touched anything related at all.

Reply:
Yeah, there was an error -- rebasing on `master` will fix this